settings.gradle

```diff
@@ -7,3 +7,4 @@ include ':magellan-sample'
 include ':magellan-sample-advanced'
 include ':magellan-support'
 include ':magellan-test'
+include ':library-test'
```

Comment:
this new module contains objects that are helpful when writing unit tests for the Magellan lib itself. It's not exposed to people who pull in the library. Thoughts on naming? I also considered nesting like we do in wf-android e.g. `:verification:library-test`